File: 250721/체질량지수/bmi.py
# 입력
intHeight, intWeight = map(int, input().split())

# 연산
intBMI = int((10000 * intWeight) / intHeight ** 2)

# 출력
print(intBMI)

# 조건문
if intBMI >= 25 :
    print("Obesity")

File: 250721/체질량지수/bmi.java
// import
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // 입력
        Scanner sc = new Scanner(System.in);
        int intHeight = sc.nextInt();
        int intWeight = sc.nextInt();

        // 연산
        int intBMI = (10000 * intWeight) / (intHeight * intHeight);

        // 출력
        System.out.println(intBMI);

        // 조건문
        if (intBMI >= 25) {
            System.out.println("Obesity");
        }
    }
}